Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
I'm getting a rather impressive mix of lib and Cellar and opt (and /usr), so I'm not sure how you usually do things. I'd just rather not handle all the autogen failures in the subpackages myself...
We did that for the standalone version (available from rpm4darwin.sourceforge.net), and I'd rather not go through it when there is a system capable of sorting out the dependencies - like brew ?